docs/content/3.providers/hygraph.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Hygraph
+description: Nuxt Image with Hygraph integration.
+links:
```

Comment:
would you add a hygraph image to the docs (`docs/public/providers/hygraph.svg`)? 🙏